src/EscapeHit.Invoice.WebApi/EscapeHit.Invoice.WebApi.csproj

```diff
@@ -4,6 +4,11 @@
     <TargetFramework>net8.0</TargetFramework>
   </PropertyGroup>
 
+  <ItemGroup>
+    <!--This is the max version supporting .NET Standard 2.0-->
```

Comment:
@viktor-kulkov At the moment we can't stick to the latest and greatest because of .NET Standard 2.0 limitation.
This is a separate topic anyway.  But in fact, lets have some architectural board and discuss upgrading to latest .NET